plugins/nessus.rb

```diff
@@ -72,10 +72,10 @@ def commands
           "nessus_index" => "Manually generates a search index for exploits",
           "nessus_template_list" => "List all the templates on the server",
           "nessus_db_scan" => "Create a scan of all IP addresses in db_hosts",
+          "nessus_db_scan_workspace" => "Create a scan of all IP addresses in db_hosts for a given workspace",
           "nessus_db_import" => "Import Nessus scan to the Metasploit connected database",
           "nessus_save" => "Save credentials of the logged in user to nessus.yml",
           "nessus_folder_list" => "List folders configured on the Nessus server",
-          "nessus_scanner_list" => "List the configured scanners on the Nessus server",
```

Comment:
did you intend to delete this command? it looks like it is still implemented otherwise.